codice/alliance
# CAL-63 Create a separate ORB service from which the NSILI Source and Endpoint will get the ORB

#### What does this PR do?
Creates a common ORB service for both the NSILI Endpoint and Source to use. 
#### Who is reviewing it?
@jaymcnallie @bdeining @kcwire 
#### How should this be tested?
Run the JUnit Tests
Build and run alliance and verify that: https://localhost:8993/services/nsili/ior.txt returns text starting with IOR:
#### Any background context you want to provide?
#### What are the relevant tickets?
https://codice.atlassian.net/browse/CAL-63
#### Screenshots (if appropriate)
#### Checklist:
